### ISSUE TITLE
Low mAP

### ISSUE THREAD

**Issue by huuthieu** [bubbliiiing/yolov4-tiny-pytorch#63]
I check your base model and Darknet yolov4-tiny. The mAP of your model is much lower than that of the Darknet model (~10%). Have you tried comparing them? And what is the configuration for the best mAP of your model?

**Comment by bubbliiiing** [comment]
In coco datasets, it's same. 0 0. Do u train your own data?

**Comment by huuthieu** [comment]
@bubbliiiing I found yolov4-tiny-3l in Darknet to give better results. Do you plan to do it?

**Comment by bubbliiiing** [comment]
three layer？like v4？

**Comment by huuthieu** [comment]
@bubbliiiing yep, you can see it's config [here](https://raw.githubusercontent.com/AlexeyAB/darknet/master/cfg/yolov4-tiny-3l.cfg)
